app/scripts/angular-apimock.js

```diff
@@ -120,6 +135,8 @@ angular.module('apiMock', [])
 						key +
 						(topLevel ? '' : ']'));
 					});
+				} else if (toSerialize === undefined || toSerialize === '') {
+					parts.push(encodeUriQuery(prefix));
```

Comment:
Yeah, I figured after i commented. But a small inline comment, linking to your comment above, would be nice. This is definitely something someone could remove by accident.